unraid_mcp/tools/users.py

```
@@ -0,0 +1,164 @@
+          me - Get current authenticated user info
+          list - List all users
+          get - Get a specific user (requires user_id)
+          add - Add a new user (requires name, password; optional role)
+          delete - Delete a user (requires user_id, confirm=True)
+          cloud - Get Unraid Connect cloud status
+          remote_access - Get remote access settings
+          origins - Get allowed origins
+        """
+        all_actions = set(QUERIES) | set(MUTATIONS)
+        if action not in all_actions:
+            raise ToolError(f"Invalid action '{action}'. Must be one of: {sorted(all_actions)}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)